packages/components/organisms/f-header/test/accessibility/axe-accessibility.spec.js

```diff
@@ -3,16 +3,28 @@ import forEach from 'mocha-each';
 const { getAxeResults } = require('../../../../../../test/utils/axe-helper');
 const Header = require('../../test-utils/component-objects/f-header.component');
 
-const header = new Header();
+let header = new Header();
 
 describe('Accessibility tests', () => {
-    forEach(['en-GB', 'en-AU', 'en-NZ', 'en-IE', 'it-IT', 'es-ES'])
-        .it('a11y - should test f-header component WCAG compliance for "%s"', tenant => {
-            // Arrange
-            header.withQuery('args', `locale:${tenant};showOffersLink:true;showDeliveryEnquiry:true`);
+    beforeEach(() => {
+        // Arrange
+        header = new Header();
+    });
 
+    forEach([
+        'en-GB',
+        'en-AU',
+        'en-NZ',
+        'en-IE',
+        'it-IT',
+        'es-ES'])
+        .it('a11y - should test f-header component WCAG compliance for "%s"', tenant => {
             // Act
-            header.load();
+            header.load({
+                locale: `${tenant}`,
```

Comment:
nit but I don't think you need the `` around tenant - guessing you can just pass in the variable so `locale: tenant` ?